### PR TITLE
No signaller in the prison LustWish

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Prisoners have access to the LustWish, but the LustWish has a signaler in it that prisoners are using to break out. This PR comments out the signaller from the lustwish.

Note that a remote signaller is already purchaseable from the Part Mart on the station, if you need one. This one specifically, however, is unrestricted to prisoners, and I'd like to rectify that. If you want me to increase the number of signallers in the Part Mart to compensate, let me know and I'll make a commit for that.

PR is tested and fully functional :)

## Why It's Good For The Game
This prevents prisoners from using the Remote Signaller to pulse wires on doors, allowing them to escape the permabrig using the LustWish contents.

## Changelog

:cl:
del: comments out the LustWish remote signaller 
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
Do you have the aptitude to make the signaler a subtype that can't be used to break out?

**Comment by ReturnToZender** [comment]
> Do you have the aptitude to make the signaler a subtype that can't be used to break out?

I could certainly try, but why would I when this is the only way to get a signaler in the prison, and you can get one by printing one or buying it from a Part-Mart

**Comment by nevimer** [comment]
> > Do you have the aptitude to make the signaler a subtype that can't be used to break out?
> 
> I could certainly try, but why would I when this is the only way to get a signaler in the prison, and you can get one by printing one or buying it from a Part-Mart outside of it

Because lustwishes don't exclusively exist in prison, and signalers are part of some play.

**Comment by ReturnToZender** [comment]
> > > Do you have the aptitude to make the signaler a subtype that can't be used to break out?
> > 
> > 
> > I could certainly try, but why would I when this is the only way to get a signaler in the prison, and you can get one by printing one or buying it from a Part-Mart outside of it
> 
> Because lustwishes don't exclusively exist in prison, and signalers are part of some play.

True, but outside of the prison, you can just buy them from the Part Mart on station

**Comment by The-Sun-In-Splendour** [comment]
Just make a lustwish subtype that doesn't have the signallers.

**Comment by YakumoChen** [comment]
> Just make a lustwish that doesn't have the signallers and put them in the prison.

I was sure this actually already exists? The seperate vendor, I mean. It just needs updated.

**Comment by Majkl-J** [comment]
> > > > Do you have the aptitude to make the signaler a subtype that can't be used to break out?
> > > 
> > > 
> > > I could certainly try, but why would I when this is the only way to get a signaler in the prison, and you can get one by printing one or buying it from a Part-Mart outside of it
> > 
> > 
> > Because lustwishes don't exclusively exist in prison, and signalers are part of some play.
> 
> True, but outside of the prison, you can just buy them from the Part Mart on station

Cafe also exists. Not sure if that can get one

**Comment by ReturnToZender** [comment]
This solution burns me but it does work as opposed to the last one.

**Review by Majkl-J — APPROVED** [review]

File: _maps/RandomRuins/LavaRuins/bubberstation/lavaland_dauntless.dmm [hunk 1]
```diff
@@ -1960,7 +1960,7 @@
 /turf/open/floor/plating,
 /area/ruin/space/has_grav/bubbers/dauntless/cargo)
 "lw" = (
-/obj/machinery/vending/dorms{
+/obj/machinery/vending/dorms/prison{
```
[the remark this answers is not in the document]
Comment (StrangeWeirdKitten) [reply _maps/RandomRuins/LavaRuins/bubberstation/lavaland_dauntless.dmm:1963]: nvm I realize now

**Pull request merged**